composer.json

```diff
@@ -21,7 +27,9 @@
         "guzzlehttp/guzzle": "~5.0",
         "roumen/feed": "~2.8",
         "thujohn/rss": "~1.0",
-        "watson/validating": "0.10.*"
+        "watson/validating": "0.10.*",
+        "maknz/slack":"dev-master",
```

Comment:
OK, I'll fork it because the master depends on Guzzle 4 and Cachet loads Guzzle 5.